crates/apub/src/objects/comment.rs

```diff
@@ -170,25 +170,29 @@ impl ApubObject for ApubComment {
       .attributed_to
       .dereference(context, local_instance(context), request_counter)
       .await?;
-    let (post, parent_comment_id) = note.get_parents(context, request_counter).await?;
+    let (post, parent_comment) = note.get_parents(context, request_counter).await?;
 
     let content = read_from_string_or_source(&note.content, &note.media_type, &note.source);
     let content_slurs_removed = remove_slurs(&content, &context.settings().slur_regex());
 
     let form = CommentForm {
       creator_id: creator.id,
       post_id: post.id,
-      parent_id: parent_comment_id,
       content: content_slurs_removed,
       removed: None,
-      read: None,
       published: note.published.map(|u| u.naive_local()),
       updated: note.updated.map(|u| u.naive_local()),
       deleted: None,
       ap_id: Some(note.id.into()),
       local: Some(false),
     };
-    let comment = blocking(context.pool(), move |conn| Comment::upsert(conn, &form)).await??;
+    // TODO Nutomic is this correct?
+    let parent_comment_path = parent_comment.map(|t| t.0.path);
+    let mut comment = blocking(context.pool(), move |conn| Comment::upsert(conn, &form)).await??;
+    comment = blocking(context.pool(), move |conn| {
+      Comment::update_ltree_path(conn, comment.id, parent_comment_path.as_ref())
+    })
+    .await??;
```

Comment:
Looks okay as far as i can tell. FYI this can be called recursively for many comments, if a deeply nested comment is fetched, and the parents arent known. There might be more efficient ways to handle it in that case (but also more complex).